packages/plugin-renderer-react/src/phenomicRendererReactClient.re

```diff
@@ -1,16 +1,16 @@
 [@bs.module "@phenomic/plugin-renderer-react/lib/client"]
 external createContainer_:
-  (ReasonReact.reactClass, Js.t({..})) => ReasonReact.reactClass =
+  (ReasonReact.reactClass, Js.t({..})) => React.element =
```

Comment:
Neither are these.

Reply:
Here indeed that's different this function take a component & return a component. So `React.component('a)` should be used instead reactClass